js/data.js

```diff
@@ -1 +1,22 @@
 /* exported data */
+var data = {
+  products: [],
+  wishlist: []
+};
+
+var xhr = new XMLHttpRequest();
+xhr.open('GET', 'https://makeup-api.herokuapp.com/api/v1/products.json?brand=maybelline');
+xhr.responseType = 'json';
+xhr.send();
+
+window.addEventListener('beforeunload', beforeUnload);
+
+function beforeUnload(event) {
+  var xhrJSON = JSON.stringify(xhr.response);
+  window.localStorage.setItem('API data', xhrJSON);
+}
+
+var previousDataJSON = localStorage.getItem('API data');
+if (previousDataJSON !== null) {
+  xhr.response = JSON.parse(previousDataJSON);
+}
```

Comment:
You shouldn't assign directly to the response property of an XHR object like this. Instead, you should have a data model where you can store this information.